app.js

```diff
@@ -25,4 +29,12 @@ stopButton.addEventListener("click", () => {
 resetButton.addEventListener("click", () => {
     clock.textContent = 0;
     seconds = 0;
-});
\ No newline at end of file
+    lapsUl.textContent = '';
+});
+
+lapButton.addEventListener('click', () => {
+    let ul = document.getElementsByTagName('ul')[0]; // AM I CORRECT IN THINKING IT HAS TO BE TAG NAME AS THIS IS A DOM PROCESS RATHER THAN HTML?
```

Comment:
try not to leave comments here, raise the PR and add the comments yourself like this.